arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -25,176 +181,375 @@ class CommonStressorParams:
         schema.description("Stressor for the benchmark workload"),
     ]
 
+    workers: typing.Annotated[
+        int,
+        schema.name("Worker Count"),
+        schema.description("Number of workers for the stressor"),
+    ]
+
 
 @dataclass
 class CpuStressorParams(CommonStressorParams):
-    cpu_count: int = field(
-        metadata={
-            "name": "CPU count",
-            "description": "Number of CPU cores to be used (0 means all)",
-        }
-    )
-    cpu_method: typing.Optional[str] = field(
-        default="all",
-        metadata={
-            "name": "CPU stressor method",
-            "description": (
-                "fine grained control of which "
-                "cpu stressors to use (ackermann, "
-                "cfloat etc."
-            ),
-        },
-    )
-
-    cpu_load: typing.Optional[int] = field(
-        default=None,
-        metadata={
-            "name": "CPU load",
-            "description": "load CPU by percentage",
-        },
-    )
+    cpu_ops: typing.Annotated[
+        typing.Optional[int],
+        schema.id("cpu-ops"),
+        schema.name("CPU Operations"),
+        schema.description("Stop CPU stress workers after N bogo operations"),
+    ] = None
+
+    cpu_load: typing.Annotated[
+        typing.Optional[int],
+        schema.id("cpu-load"),
+        schema.name("CPU Load"),
+        schema.description(
+            "Load CPU with P percent loading for the CPU stress workers"
```

Comment:
There's no referent here for "P" ... that's not really an improvement over the old wording. You could use the parameter name, like "`<cpu_load>` percent" or something like "specify the target percentage CPU load". (And, is this cumulative across the set of CPU stress workers, or, more likely, applicable to each worker for its target CPU? That is, I does `workers: 2, cpu_load: 80` mean each of two workers should target to consume 80% of their individual CPUs? If so, "Target CPU load percentage for each CPU stress worker" might be better overall phrasing...

Reply:
I second the notion that it would be better to use normal English than to use random letters.  (There are more instances of this in `VmStressorParams`.)  If you don't like Dave's suggestions, you can go with something like this (with words elided as appropriate):
> This parameter specifies the target CPU load to be imposed by the workers as a percentage utilization.

Similarly, below:
> [This parameter specifies the] number of bytes of shared memory created by each worker; the default is 256MB.

Etc.

Reply:
To be fair, most if not all of the description updates I made in the PR came copy-paste from the stress-ng man page, but I generally only copied in the first sentence of long descriptions, so some context is lost. True, we can argue to document better than the source project, but that might be out-of-scope for this PR.

Reply:
Ha: I kinda think that generally one is on shaky ground after changing the wording of strings to argue that changes to the wording of those strings might be out of scope. 😆

And, regardless of what stressng says, (increasingly convinced it's not to be held up as any sort of "benchmark" of documentation quality), I'm really unthrilled with referring to a parameter by a token that's not the name of the parameter and doesn't appear otherwise in the description. Webb's "This parameter specifies" would be an improvement if you don't want to repeat the parameter token.